frontend/app/src/components/MonacoDiffView.vue

```diff
@@ -78,6 +78,15 @@ export default {
               this.foldAll(editor);
             }
             this.jumpActions(editor);
+
+            var labelCssClass = [ 'bugfix', 'whitespace','documentation', 'refactoring', 'test', 'unrelated'];
+            for(var i = 0; i < this.file.lines.length; i++)
+            {
+              var line = this.file.lines[i];
+              var label = labelCssClass[line.label];
```

Comment:
This would set bugfix for label 0 but it should be 1. While setting everything not pre-labeled to bugfix would be the status quo we should define 0 from the backend to not labeled.

Reply:
Yeah... updated the code